pkg/apis/pravega/v1alpha1/status.go

```diff
@@ -22,6 +22,11 @@ const (
 	ClusterConditionPodsReady ClusterConditionType = "PodsReady"
 	ClusterConditionUpgrading                      = "Upgrading"
 	ClusterConditionError                          = "Error"
+
+	// Reasons for cluster upgrading condition
+	UpgradingControllerReason   = "UpgradingController"
```

Comment:
I think it would be good to have the string as "Updating Controller" since same string can be used for upgrade as well as rollback on the component.